### ISSUE TITLE
Link '/notes/mine' to my own notes

### ISSUE THREAD

**Issue by pietervdvn** [openstreetmap/openstreetmap-website#1997]
Hi,

It would be very convenient to have 'osm.org/notes/mine' refer to my (the logged in user) notes - just like osm.org/traces/mine.

Is this possible?

**Comment by tuckerrc** [comment]
Implemented in #2699

**Comment by pietervdvn** [comment]
:partying_face: 

Thanks for implementing! Looking forward to the deploy

**Comment by tomhughes** [comment]
It was deployed seven months ago.

**Comment by pietervdvn** [comment]
Then I'm missing something: https://www.openstreetmap.org/notes/mine does not give a result. Maybe the URL was changed?

**Comment by tomhughes** [comment]
No idea. I just know that anything merged last July is long since deployed.

**Comment by tomhughes** [comment]
Ah it wasn't actually merged, rather #2699 was, but that doesn't actually implement `/notes/mine`.

**Comment by tuckerrc** [comment]
:facepalm: I misread the pull request. Sorry about that

**Comment by pietervdvn** [comment]
No problem, that happens ;)

**Comment by tuckerrc** [comment]
The /traces/mine route redirects to /user/[username]/traces which is what you need I think. You can go to /user/[username]/notes to see all the notes uploaded by the user. Do we want to create a redirect for /notes/mine as well?

**Comment by pietervdvn** [comment]
Yes, that was the intent of this issue, to simply have a redirect to `/user/[username]/notes`
